chapters/annotations.tex

```diff
@@ -39,14 +39,19 @@ \section{Documentation}\label{annotations-for-documentation}\label{documentation
 The \fmtannotationindex{Documentation} annotation has the following contents, where the \lstinline!info! and \lstinline!revisions! annotations are described in \cref{annotation-info-revisions}, and the \lstinline!figures! annotation is described in \cref{annotations-for-figures}:
 \begin{lstlisting}[language=modelica]
 record Documentation
-  String info = "" "Description of the class";
-  String revisions = "" "Revision history";
+  constant String info = "" "Description of the class";
+  constant String revisions = "" "Revision history";
   Figure[:] figures = {}; "Simulation result figures";
 end Documentation;
 \end{lstlisting}
 
 How the tool interprets the information in \lstinline!Documentation! is unspecified.
 
+\subsection{How to interpret annotation definitions}
+If the annotation itself is described by a \lstinline!record! \emph{class} the annotation can be seen as a modifier for a \lstinline!record! \emph{component} with the same name.
+If the annotation is declared as a parameter the corresponding modifier must be an evaluable parameter.
+If the annotation is declared as a constant the corresponding modifier must be a literal value.
```

Comment:
Again, it would be good to avoid writing just `constant` when we mean something different than constant variability.  For instance:
```suggestion
If the annotation is declared as \lstinline!constant! the corresponding modifier must be a constant expression, or a literal value in case the \lstinline!constant! keword is prefixed with $\mathit{literal}$, as in \lstinline!$\mathit{literal}$ constant Boolean singleInstance!.
```

Reply:
Looks good now.  Resolving.